adafruit_ble/__init__.py

```diff
@@ -53,46 +54,60 @@
 __repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_BLE.git"
 
 class BLEConnection:
-    """This represents a connection to a peer BLE device.
-
-       It acts as a map from a Service type to a Service instance for the connection.
     """
-    def __init__(self, connection):
-        self._connection = connection
-        self._discovered_services = {}
-        """These are the bare remote services from _bleio."""
+    Represents a connection to a peer BLE device.
+    It acts as a map from a `Service` type to a `Service` instance for the connection.
 
+    :param bleio_connection _bleio.connection
```

Comment:
This doc looks incomplete.

Reply:
Most of the doc is in the method docs.

Reply:
Let me know if you want more and what it would be.